README.md

```diff
@@ -36,17 +36,26 @@ jobs:
           echo "Running test 1..."
       - name: Deploy With Value 2
         uses: "infinite-automations/terraform-all-in-one@v1.1.0"
-        with:          
+        with:
           directory: "./terraform"
           apply: true
         env:
           TF_VAR_my_var: "value2"
       - name: Run Test 2
         run: |
           echo "Running test 2..."
+      - name: Run Test 3
+        uses: "infinite-automations/terraform-all-in-one@v1.1.0"
+        with:
+          directory: "./terraform"
+          test-directory: "./terraform"
+          format: false
+          validate: false
+          plan: false
+          test: true
```

Comment:
```suggestion
      - name: Run Test 2
        uses: "infinite-automations/terraform-all-in-one@v1.1.0"
        with:
          directory: "./terraform"
          test-directory: "./terraform"
          format: false
          validate: false
          plan: false
          test: true
```

Reply:
and do not forget to increase `numMatches` and `numReplacements` in the [release config](https://github.com/infinite-automations/terraform-all-in-one/blob/add-tf-test/.releaserc.json)